DataFiles/reference-documents.csv

```diff
@@ -536,10 +536,12 @@ EN ISO 8394-1:2010,2010-10-01,Building construction - Jointing products - Part 1
 EN ISO 8394-2:2017,2017-11-22,Buildings and civil engineering works - Determination of extrudability for sealant - Part 2: Using standardized apparatus (ISO 8394-2:2017)
 EN ISO 9001,1900-01-01,quality management systems - requirements
 EN ISO 9053-1:2018,2018-12-12,Acoustics - Determination of airflow resistance - Part 1: Static airflow method (ISO 9053-1:2018)
+European Illustrated Glossary of Conservation Terms for Wall Paintings and Architectural Surfaces,2015-01-01,EwaGlos – European Illustrated Glossary of Conservation Terms for Wall Paintings and Architectural Surfaces
```

Comment:
"European Illustrated Glossary of Conservation Terms for Wall Paintings and Architectural Surfaces" is not a code and it's too long. Now it is the same as the description. Suggestion: EIGCTWPAS-2015.
Codes are meant for efficient digital communication, not to be human readable :-)